api/envoy/api/v2/cds.proto

```diff
@@ -344,6 +344,19 @@ message Cluster {
       DEFAULT_SUBSET = 2;
     }
 
+    // If NOT_DEFINED top level config fallback policy is used instead.
+    // If NO_FALLBACK is selected, a result
+    // equivalent to no healthy hosts is reported. If ANY_ENDPOINT is selected,
+    // any cluster endpoint may be returned (subject to policy, health checks,
+    // etc). If DEFAULT_SUBSET is selected, load balancing is performed over the
+    // endpoints matching the values from the default_subset field.
+    enum LbSubsetSelectorFallbackPolicy {
+      NOT_DEFINED_PER_SELECTOR = 0;
```

Comment:
I think the `_PER_SELECTOR` suffix is a bit verbose/redundant here, or maybe I'm missing something, LMK.

Reply:
I see, this is because we already have these values above for the top-level. Can you nest this `enum` in `LbSubsetSelector`? It probably belongs there anyway.

Reply:
After working on doc part, agree that is less confusing from user perspective to get rid of `_PER_SELECTOR` and is now possible after moving enum into `LbSubsetSelector`.